plugins/outputs/whatap/whatap.go

```diff
@@ -0,0 +1,168 @@
+package whatap
+
+import (
+	"fmt"
+	"net"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/plugins/outputs"
+
+	whatap_io "github.com/whatap/go-api/common/io"
+	whatap_pack "github.com/whatap/go-api/common/lang/pack"
+	whatap_hash "github.com/whatap/go-api/common/util/hash"
```

Comment:
Please do not use dashes as this seems alien in Golang. Either use `whatapIo` etc or shorten to `wio`, `wpack` and `whash`...

Reply:
Changed  to wio, wpack, whash